Core/include/Acts/Geometry/Extent.hpp

```diff
@@ -181,8 +198,9 @@ class Extent {
   bool constrains(BinningValue bValue = binValues) const;
 
   /// Convert to output stream for screen output
-  /// @param sl [in,out] The output stream
-  std::ostream& toStream(std::ostream& sl) const;
```

Comment:
is this completely obsolete or should we keep it and implement `toString` with he help of this function?